resources/definitions/matterhacker_pulsexe.def.json

```diff
@@ -0,0 +1,231 @@
+	    },
+	    "machine_max_feedrate_x":{
+	        "value":250
+	    },
+	    "machine_max_feedrate_y":{
+	        "value":250
+	    },
+	    "machine_max_feedrate_z":{
+	        "value":10
+	    },
+	    "machine_max_feedrate_e":{
+	        "value":50
+	    },
+	    "machine_max_acceleration_x":{
+	        "value":250
+	    },
+	    "machine_max_acceleration_y":{
+	        "value":250
+	    },
+	    "machine_max_acceleration_z":{
+	        "value":100
+	    },
+	    "machine_max_acceleration_e":{
+	        "value":5000
+	    },
+	    "machine_acceleration":{
+	        "value":500
+	    },
+	    "machine_max_jerk_xy":{
+	        "value":10
+	    },
+	    "machine_max_jerk_z":{
+	        "value":0.4
+	    },
+	    "machine_max_jerk_e":{
+	        "value":5
+	    },
+	    "machine_endstop_positive_direction_x":{
+	        "default_value":true
+	    },
+	    "machine_endstop_positive_direction_y":{
+	        "default_value":true
+	    },
+	    "machine_heated_bed":{
+	        "default_value":true
+	    },
+	    "material_diameter":{
+	        "default_value":1.75
+	    },
+	    "acceleration_print":{
+	        "value":500
+	    },
+	    "acceleration_travel":{
+	        "value":1300
+	    },
+	    "acceleration_travel_layer_0":{
+	        "value":"acceleration_travel"
+	    },"acceleration_roofing":{
+	        "enabled":"acceleration_enabled and roofing_layer_count > 0 and top_layers > 0"
+	    },
+	    "jerk_print":{
+	        "value":8
+	    },
+	    "jerk_travel":{
+	        "value":"jerk_print"
+	    },
+	    "jerk_travel_layer_0":{
+	        "value":"jerk_travel"
+	    },
+	    "acceleration_enabled":{
+	        "value":false
+	    },
+	    "jerk_enabled":{
+	        "value":false
+	    },
```

Comment:
```suggestion
```
DRY Principle. These are already defaults, so there is no need to re define them here.